### DISCUSSION TITLE
error - ubuntu-latest pipelines will use ubuntu-24.04 soon

### DISCUSSION THREAD

**Discussion by Donostio** [cecobask/imdb-trakt-sync#68] [General]
1 error and 1 warning
sync
Process completed with exit code 2.
sync
ubuntu-latest pipelines will use ubuntu-24.04 soon. For more details, see https://github.com/actions/runner-images/issues/10636

**Comment by cecobask** [comment]
Hey @Donostio, thanks for reporting.
The failure in your sync isn't recurring. I can see that it resolved itself in subsequent runs.
I fixed the Ubuntu version to `24.04` in the GitHub Actions workflows of the repository.
The warning should be gone once you sync your fork.
Happy holidays! 🎅

**Comment by Donostio** [comment]
Awesome, thanks you so much for all your help, and for kindly making this
available and all the hard work keeping it going.
